client/e2e/support/user-list.po.ts

```diff
@@ -0,0 +1,40 @@
+export class UserListPage {
+  navigateTo() {
+    return cy.visit('/users');
+  }
+
+  getUrl() {
+    return cy.url();
+  }
+
+  getUserTitle() {
+    return cy.get('.user-list-title');
+  }
+
+  getUserCards() {
+    return cy.get('.user-cards-container app-user-card');
+  }
+
+  getUserListItems() {
+    // return element(by.className('user-nav-list')).all(by.className('user-list-item'));
+    return cy.get('.user-nav-list .user-list-item');
+  }
+
+  clickViewProfile(card: Cypress.Chainable<JQuery<HTMLElement>>) {
+    // return card.element(by.buttonText('VIEW PROFILE')).click();
+    return card.contains('button', 'VIEW PROFILE').click();
+  }
+
+  changeView(viewType: 'card' | 'list') {
+    return cy.get(`#view-type-radio .mat-radio-button[value="${viewType}"]`).click();
+  }
+
+  selectRole(value: string) {
```

Comment:
Should this be an OR type with the set of legal values for the `string` here?

Reply:
I have switched this to using the `UserRole` type from the app.